imagersite/imager_profile/tests.py

```diff
@@ -0,0 +1,48 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+from imager_profile.models import ImagerProfile
+
+import factory
+# Create your tests here.
+
+class UserFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        """."""
+        model = User
+    username = factory.Sequence(lambda n: "user{}".format(n))
+    email = factory.Sequence(
+        lambda n: "user{}@example.com".format(n)
+    )
+
+
+class ProfileTestCase(TestCase):
+    """Test for the profile model."""
+    def setUp(self):
+        users = [UserFactory.create() for i in range(20)]
+
+        for user in users:
+            user.set_password('foo')
+            user.save()
+
+        self.users = users
+
+    def test_every_profile_must_have_a_user(self):
+        """Test for profile instances."""
+        with self.assertRaises(Exception):
+            patron = ImagerProfile()
+            patron.save()
+
+    def test_profile_with_user_prints_username(self):
+        """."""
+        some_profile = ImagerProfile.objects.first()
+        self.assertTrue(str(some_profile), some_profile.user.username)
+
+    def test_new_user_has_a_profile(self):
+        """."""
+        user = UserFactory.create()
+        profile = ImagerProfile.objects.last()
+        self.assertTrue(profile.user == user)
+
+    def test_there_are_as_many_users_in_profile(self):
+        """."""
+        self.assertEquals(len(User.objects.all()), len(ImagerProfile.objects.all()))
```

Comment:
You should add more tests. 

try:
 - when you delete a profile does it delete the user?
- when you do ImagerProfile.active, does it give you a list of all the active users.
- test your string display method, it should run in both python 2 and 3
- test that when you update some of the fields in your profile and save they update in the db.